ios/Podfile

```diff
@@ -37,10 +37,19 @@ end
 
 # Fix a open bug in react-native-config
 # @see https://github.com/luggit/react-native-config/issues/125
+# (using a fix proposed in one of that issue's comments)
 post_install do |installer|
   installer.pods_project.targets.each do |target|
-    if target.name == "React"
-      target.remove_from_project
+    if target.name == 'react-native-config'
```

Comment:
Not neded duplicate of https://github.com/teamdigitale/italia-app/blob/master/bin/add-ios-env-config.sh